common/events.ts

```diff
@@ -0,0 +1,16 @@
+export type ContractCardView = {
+  name: 'view market'
+  slug: string
+  timestamp: number
+  contractId: string
+  creatorId: string
+  groupSlugs: string[]
+}
+
+export type ContractView = {
+  name: 'view market card'
```

Comment:
The `name` strings here seem to be swapped.

Reply:
good eye, thanks!